exercises/react/example.cr

```diff
@@ -0,0 +1,71 @@
+alias CellValue = Int32
+
+private class Cell
+  @dependents = [] of ComputeCell
+
+  def initialize(initial : CellValue)
+    @value = initial
+  end
+
+  getter :value
+
+  protected getter :dependents
+  protected def propagate; _propagate; end
```

Comment:
it was *probably* because `ComputeCell` was a private class, and now it is not. I will test a few things to confirm.